src/datasets/jetstream.md

```diff
@@ -0,0 +1,243 @@
+so on.
+
+Clients are given a fixed amount of time, specified in Experimenter and
+often a week long, to enroll. Final day 0 results are available for
+reporting at the end of the enrollment period, after the last eligible
+client has enrolled, and week 0 results are available a week after the
+enrollment period closes. Results for each window are published as soon
+as complete data is available for all enrolled clients.
+
+The “overall” window, published after the experiment has ended, is a
+window beginning on each client’s day 0 that spans the longest period
+for which all clients have complete data.
+
+Jetstream computes statistics over several metrics by default, including
+for any features associated with the experiment in Experimenter. Data
+scientists can provide configuration to add additional metrics. Advice
+on configuring Jetstream can be found at Jetstream’s Github wiki.
+
+# Statistics tables
+
+The statistics tables contain statistical summaries of their
+corresponding aggregate tables. These tables are suitable for plotting
+directly without additional transformations.
+
+Statistics tables are named like:
+
+`statistics_<slug>_{day, week, overall}_<index>`
+
+A view is also created that concatenates all statistics tables for an
+experiment of a given period type, named like:
+
+`statistics_<slug>_{daily, weekly, overall}`
+
+Statistics tables have the schema:
+
+<table>
+<thead>
+<tr class="header">
+<th>Column name</th>
+<th>Type</th>
+<th>Description</th>
+</tr>
+</thead>
+<tbody>
+<tr class="odd">
+<td>`segment`</td>
+<td>STRING</td>
+<td>The segment of the population being analyzed. “all” for the entire population.</td>
+</tr>
+<tr class="even">
+<td>`metric`</td>
+<td>STRING</td>
+<td>The slug of the metric, like `active_ticks` or `retained`</td>
+</tr>
+<tr class="odd">
+<td>`statistic`</td>
+<td>STRING</td>
+<td>The slug of the statistic that was used to summarize the metric, like “mean” or “deciles”</td>
+</tr>
+<tr class="even">
+<td>`parameter`</td>
+<td>NUMERIC (decimal)</td>
+<td>A statistic-dependent quantity. For two-dimensional statistics like “decile,” this represents the x axis of the plot. For one-dimensional statistics, this is NULL.</td>
+</tr>
+<tr class="odd">
+<td>`comparison`</td>
+<td>STRING</td>
+<td>If this row represents a comparison between two branches, this row describes what kind of comparison, like `difference` or `relative_uplift`. If this row represents a measurement of a single branch, then this column is NULL.</td>
+</tr>
+<tr class="even">
+<td>`comparison_to_branch`</td>
+<td>STRING</td>
+<td>If this row represents a comparison between two branches, this row describes which branch is being compared to. For simple A/B tests, this will be “control.”</td>
+</tr>
+<tr class="odd">
+<td>`ci_width`</td>
+<td>FLOAT64</td>
+<td>A value between 0 and 1 describing the width of the confidence interval represented by the lower and upper columns. Valued at 0.95 for 95% CIs.</td>
```

Comment:
I think we can just use "confidence interval", rather than the abbreviation. Although... maybe I'm just revealing my ignorance but I find the use of "width" here confusing. Is this referring to a confidence level?

```suggestion
<td>A value between 0 and 1 describing the width of the confidence interval represented by the lower and upper columns. For example, 0.95 represents a confidence interval of 95%.</td>
```

Reply:
Confidence level is correct, yes. (I try to avoid "confidence level" to avoid suggesting that the reader should feel subjectively confident.)

Reply:
Got it. I'd still prefer to avoid the abbreviation, but I'll leave the decision to you.

Reply:
Oh, I'd made the change :)